### PR TITLE
Include level and runner in pronto messages

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

File: lib/gooddata.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ module GoodData
 require_relative 'gooddata/exceptions/exceptions'
 require_relative 'gooddata/helpers/helpers'
 require_relative 'gooddata/lcm/lcm'
-require_relative 'gooddata/lcm/lcm2'
+require_relative 'gooddata/lcm/lcm2'
```
Comment (yenkins) [inline lib/gooddata.rb]: warning: Style/TrailingWhitespace: Trailing whitespace detected. (rubocop)

File: lib/gooddata/client.rb [hunk 2]
```diff
@@ -64,4 +64,4 @@ def release_info
 end
 
 # Init requires
-GoodData.init_module
+GoodData.init_module
```
Comment (yenkins) [inline lib/gooddata/client.rb]: warning: Style/TrailingWhitespace: Trailing whitespace detected. (rubocop)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/482/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1313/ : FAILURE in 1m 06s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1583/ : SUCCESS in 1m 04s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/484/ : SUCCESS in 5s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1315/ : SUCCESS in 1m 06s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1585/ : SUCCESS in 1m 07s

**Pull request merged**